lib/components/src/typography/DocumentFormatting.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import { styled, CSSObject } from '@storybook/theming';
+import { window } from 'global';
```

Comment:
Is this correct? I didn't know this pattern.

Reply:
That's fine.